conda/_vendor/appdirs.py

```diff
@@ -10,6 +10,9 @@
 #   http://support.microsoft.com/default.aspx?scid=kb;en-us;310294#XSLTH3194121123120121120120
 # - Mac OS X: http://developer.apple.com/documentation/MacOSX/Conceptual/BPFileSystem/index.html
 # - XDG spec for Un*x: http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
+from ..deprecations import deprecated
+deprecated.module("23.11", "24.5", addendum="Use `platformdirs` instead.")
```

Comment:
```suggestion
deprecated.module("24.3", "24.9", addendum="Use `platformdirs` instead.")
```

Reply:
D'oh, I always stumble over the UX of the deprecation helper, that the first number is not when we start the pending deprecation, but the actual deprecation.